bin/download_cartopy_data.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+import argparse
+import os
+import urllib.request
+import tempfile
+import cartopy
+from cartopy import config
+import sys
+
+
+def main(target_dir):
+    try:
+        os.makedirs(target_dir)
+    except OSError:
+        pass
+
+    version = cartopy.__version__
+
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        target_dir = os.path.abspath(target_dir)
+        owd = os.getcwd()
+        os.chdir(tmpdirname)
+        urllib.request.urlretrieve(
+            'https://raw.githubusercontent.com/SciTools/cartopy/v{}/tools/feature_download.py'.format(version),
+            'feature_download.py')
+
+        file = open('__init__.py', 'w')
+        file.close()
+
+        sys.path.append(tmpdirname)
+
+        from feature_download import FEATURE_DEFN_GROUPS, download_features
```

Comment:
Here's the part I'm not thrilled about.  @willirath, any idea how to do this better?